examples/Devices/ErgoDox/Makefile

```diff
@@ -0,0 +1,55 @@
+# This stub makefile for a Kaleidoscope example pulls in all the targets
```

Comment:
If you wouldn't mind pulling out the stub makefiles to their own PR, that'd help reduce the amount of unrelated churn in this PR.

Reply:
This didn't actually happen.

Reply:
Ow, yeah, sorry. I pulled all the _other_ Makefiles out, and missed this one. Dropping in a bit.